### PR TITLE
Explicitly state request rate in docs

### PR DESCRIPTION
**Pull request by davidxia**
for both endpoint-request-rate and dropped-request-rate.

### CODE REVIEW

File: modules/metrics/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ A Meter metric, per status-code, tagged with:
 |-------------|----------------------------|------------------------------------------------------|
 | what        | "endpoint-request-rate"    | Enable/disable with ENDPOINT_REQUEST_RATE            |
 | status-code | *                          | "200", "404", "418", etc.                            |
-| unit        | "request"                  |                                                      |
+| unit        | "request/s"                | Requests per second.                                 |
```
Comment (davidxia) [inline modules/metrics/README.md]: I noticed that internally for us this is "message/s". Am I correct in assuming the open-source module without Spotify-flavoring will have "request/s" as its unit?

**Comment by davidxia** [comment]
@pettermahlen

File: modules/metrics/README.md (hunk 1 above)
Comment (pettermahlen) [inline modules/metrics/README.md]: The `endpoint-request-rate` metric generated by internal Apollo in fact has the unit 'request', exactly like here. I think 'request/s' is a better unit, and slightly better than 'message/s'?

**Comment by pettermahlen** [comment]
I think when making this change, we should also update the units used in the actual metrics tags as per the `SemanticServiceMetrics` class. Would you be able to add that? There seems to be other rates that have incorrect units (like the dropped requests rate, for instance).

**Comment by jo-ri** [comment]
"/s" is appended to the 'unit' tags by ´FastForwardReporter´ so they should be defined without it in ´ SemanticServiceMetrics´.

**Comment by davidxia** [comment]
@jo-ri Why does FastForwardReporter handle the appending of the "/s"? Why not just have `SemanticServiceMetrics` do that?

**Comment by jo-ri** [comment]
@davidxia I don't know, I haven't worked with FastForwardReporter. Maybe @udoprog can shed some light?

**Comment by udoprog** [comment]
Hey,

When some things are registered the system actually exposes multiple different things, sometimes with different units.
The most notable example is timer, which exposes histograms and meters. To make things consistent we made it so that the MetricId would only contain the singular unit and we derive the unit for the rates as described.

I'm open for discussing if this is the 'right' thing or not.

**Comment by davidxia** [comment]
Hm, I don't know enough to have an opinion. As a user, I just wanted the docs to be more specific. Does this PR make sense anymore or should we close as the changes seem to be incorrect?

**Comment by pettermahlen** [comment]
It does make sense, and that my suggestion of also updating the code was wrong. :) Merging.

**Pull request merged**